docker-compose.yml

```diff
@@ -15,6 +15,7 @@ services:
       - "/run/dbus/system_bus_socket:/run/dbus/system_bus_socket"
       - "dappmanagerdnpdappnodeeth_data:/usr/src/app/dnp_repo/"
       - "/usr/src/dappnode/DNCORE/:/usr/src/app/DNCORE/"
+      - "/etc/systemd/system/:/usr/src/app/DNCORE/services/host/"
```

Comment:
@3alpha Do you see any security issues with mapping this path?

Reply:
The most dangerous thing I have found is to delete the docker volume, that would remove the host services which is potentially dangerous